src/Npgsql/NpgsqlParameter.cs

```diff
@@ -245,9 +244,7 @@ public NpgsqlParameter(string parameterName, DbType parameterType, int size, str
             {
                 // ReSharper disable once ConditionIsAlwaysTrueOrFalse
                 if (value == null)
-                {
                     _name = TrimmedName = string.Empty;
-                }
                 else if (value.Length > 0 && (value[0] == ':' || value[0] == '@'))
                 {
                     TrimmedName = value.Substring(1);
```

Comment:
Would you mind to remove braces here too? (:

```csharp
TrimmedName = (_name = value).Substring(1));
```

Reply:
Done